.github/workflows/action.yml

```diff
@@ -22,7 +22,6 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Extract tag from github.ref
-        id: docker-tag
```

Comment:
We already re-added that id as the output needs to be addressed for the docker tag